apps/backend/src/db/migrations/20250727185929_plans_days_task_tables.ts

```diff
@@ -65,7 +65,6 @@ async function up(knex: Knex): Promise<void> {
 		table.string(PlanColumnName.TITLE, ColumnLength.TITLE).notNullable();
 		table
 			.integer(PlanColumnName.USER_ID)
-			.notNullable()
```

Comment:
We should not modify old migrations to avoid inconsistency. Do it in a new migration